catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/EmailSender.java

```diff
@@ -0,0 +1,103 @@
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.alliance.nsili.endpoint;
+
+import static org.apache.commons.lang3.Validate.notNull;
+
+import java.util.Properties;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// TODO: 9/2/16 make into interface 
+public class EmailSender {
+
+    private static final String SMTP_HOST_PROPERTY = "mail.smtp.host";
+    private static final Logger LOGGER = LoggerFactory.getLogger(EmailSender.class);
+    private String fromEmail;
+    private String mailHost;
+
+    public EmailSender(String fromEmail, String mailHost) {
+        notNull(fromEmail, "fromEmail must be non-null");
+        notNull(mailHost, "mailHost must be non-null");
+
+        this.fromEmail = fromEmail;
+        this.mailHost = mailHost;
+    }
+
+    public void sendEmail (String emailDest) {
+        try {
+            String subject = "Test email";
+            String emailBody = "Body of email to test";
+
+            InternetAddress emailAddr = new InternetAddress(emailDest);
+            emailAddr.validate();
+
+            Properties properties = createSessionProperies();
+
+            Session session = Session.getDefaultInstance(properties);
+            MimeMessage mimeMessage = new MimeMessage(session);
+            mimeMessage.setFrom(new InternetAddress(fromEmail));
+            mimeMessage.addRecipient(Message.RecipientType.TO, emailAddr);
+            mimeMessage.setSubject(subject);
+            mimeMessage.setText(emailBody);
+            Transport.send(mimeMessage);
+
+        } catch (AddressException e) {
+            LOGGER.error("Invalid email address entered", e);
+        } catch (MessagingException e) {
+        }
+    }
+
+    private Properties createSessionProperies() {
+        Properties properties = System.getProperties();
+        properties.setProperty(SMTP_HOST_PROPERTY, mailHost);
+        return properties;
+    }
+
+    public String toString() {
+        return "EmailSender{" +
+                "fromEmail=" + fromEmail + '\'' +
+                ", mailHost=" + mailHost +
+                '}';
+    }
+
+    // needs 'unused' tag?
+    public String getFromEmail() {
+        return fromEmail;
+    }
+
+    public void setFromEmail(String fromEmail) {
+        this.fromEmail = fromEmail;
+    }
+
+    public String getMailHost() {
+        return mailHost;
+    }
+
+    public void setMailHost(String mailHost) {
```

Comment:
I would add javadoc indicating that mailHost shouldn't be null and add a 'notNull' check in the code.